bin/diffeco

```diff
@@ -31,10 +31,18 @@ done
 
 BASEDIR=$(dirname "$0")
 
+# We need a GNU tar, so try gtar first
+which gtar 2>1 > /dev/null
```

Comment:
On Linux this leaves an empty file `1` on the file system.

Reply:
Oops, that's a typo. Fixed in the new commit (hopefully). Let me know!